### PR TITLE
Feat: Avatar and badge components

### PR DESCRIPTION
**Pull request by Vishruta-Patil**

### CODE REVIEW

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+@import url("../../style/reset.css");
+@import url("../../style/color.css");
+@import url("../../style/common.css");
+
+.avatar{
+  border-radius: 50%;
+  width: 48px;
```
Comment (Jia-A) [inline Components/Avatar/avatar.css:7]: change it into rem.

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+.sm {
+  width: 32px;
+  height: 32px;
+  font-size: 1rem;
+}
+
+.md {
+  width: 48px;
+  height: 48px;
```
Comment (Jia-A) [inline Components/Avatar/avatar.css:25]: px to rem

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  font-size: 2rem;
+}
+
+.lg {
+  width: 64px;
+  height: 64px;
```
Comment (Jia-A) [inline Components/Avatar/avatar.css:31]: px to rem

File: Components/Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+.badge-xl {
+  width: 18px;
+  height: 18px;
+  bottom: 2px;
+  right: 2px;
+  border: 3px solid white;
+}
+.badge-lg {
+  width: 15px;
+  height: 15px;
```
Comment (Jia-A) [inline Components/Avatar/avatar.css:85]: px to rem

File: Components/Avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  width: 25px;
+  height: 25px;
+  bottom: 5px;
+  right: 5px;
+  border: 3px solid white;
+}
+
+.badge-xl {
+  width: 18px;
+  height: 18px;
```
Comment (Jia-A) [inline Components/Avatar/avatar.css:78]: px to rem

**Review by Jia-A — COMMENTED** [review]
You might want to change all the values from px to rems, it would make your site a bit more responsive. Your avatar.js file is empty. Everything else looks good to me.

**Review by nikhilmalviya-dev — APPROVED** [review]
good work but replace px into rem.

File: Components/Avatar/avatar.css (hunk 2 above)
Comment (Bharatrathod-81) [reply Components/Avatar/avatar.css:25]: It would be good if you change the px into rem for the responsiveness

**Review by Bharatrathod-81 — APPROVED** [review]

**Comment by Vishruta-Patil** [comment]
Thanks, @Jia-A @nikk422 @Bharatrathod-81 for the review.

**Pull request merged**